app/src/main/java/com/cricketchirps/MainActivity.java

```diff
@@ -12,6 +18,56 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        Button button = findViewById(R.id.btn_submit);
+        Button button_submit = findViewById(R.id.btn_submit);
+        final RadioButton radioBtn_celsius = findViewById(R.id.radioBtn_celsius);
+        final RadioButton radioBtn_fahrenheit = findViewById(R.id.radioBtn_fahrenheit);
+        final TextView textView_temperature = findViewById(R.id.textView_temperature);
+        final EditText txtBx_data =findViewById(R.id.txtBx_data);
+        final Toast toast_fahrenheit = Toast.makeText(getApplicationContext(),"Count the number of chirps in 14 seconds",Toast.LENGTH_SHORT);
+        final Toast toast_celcious = Toast.makeText(getApplicationContext(),"Count the number of chirps in 25 seconds",Toast.LENGTH_SHORT);
```

Comment:
Could be better if you put your hardcoded to strings file for make the string easier to change, what do you think about?